x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/constants.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ANY_AUTH_PROVIDER = '*';
+
+export const ANY_AUTH_PROVIDER_OPTION_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.roleMapping.any.label',
```

Comment:
Can you see if you can make these labels match up with the i18n guide? https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

So like...

`'xpack.enterpriseSearch.roleMapping.any.label'` -> `'xpack.enterpriseSearch.roleMapping.any DropDownOptionLabel'`

`'xpack.enterpriseSearch.roleMapping.addRoleMapping.button'` -> `'xpack.enterpriseSearch.roleMapping.addRoleMappingButtonLabel'`